src/app/workflow/snapshot-exporter-modal/snaphot-exporter-modal.component.scss

```diff
@@ -0,0 +1,37 @@
+@import 'materialColorScheme.scss';
+
+.confirmation-box {
+  margin: 0 0 9px;
+  padding: 8px 0.3px 8px 10.7px;
+  border-radius: 8px;
+  border: solid 1px #fff176;
+  background-color: #fffde7;
```

Comment:
random color?

Reply:
From Zepplin/mock; nothing close in palette, unless we bring what's called "warning" in the color paletter (not $kim-warn). Leaving for now.